src/AppBundle/Controller/BaptismController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Baptism;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use DateTime;
+
+/**
+ * Baptism controller.
+ *
+ */
+class BaptismController extends Controller
+{
+
+    /**
```

Comment:
In order to have a well documented project, can you add comments to explain the action purpose please ?